src/api/app/views/webui/project/show.html.erb

```diff
@@ -109,7 +109,11 @@
                 <%= link_to(sprited_text('tools-report-bug', 'Report bug'), bugzilla_url(@bugowners_mail, "#{@project.name}: Bug")) -%>
               </li>
           <% end -%>
-          <% if User.current.can_modify_project?(@project, true) %>
+          <% if @project.is_locked? && User.current.can_modify_project?(@project, true) %>
+                  <li>
+                    <%= link_to(sprited_text('lock_open', 'Unlock project'), { controller: 'project', action: 'unlock_dialog', project: @project.name }, remote: true) %>
+                  </li>
+          <% elsif User.current.can_modify_project?(@project) %>
```

Comment:
no, it needs to run the can_modify check with and without the ignoreLock option (the true as second argument)